### ISSUE TITLE
Loss of granularity in recent vocabulary change

### ISSUE THREAD

**Issue by gowthamrao** [OHDSI/Vocabulary-v5.0#1004]
I want to build a cohort of persons who had lymph node positive disease in some lymph nodes. This is my current concept set expression

![image](https://github.com/OHDSI/Vocabulary-v5.0/assets/13936600/492464eb-468e-4138-8a3b-2693cf839ac0)

In the version v20240229 of vocabulary these conceptIds have become non - standard. 
![image](https://github.com/OHDSI/Vocabulary-v5.0/assets/13936600/e0c55240-f9c4-4c51-921b-7c7b93671517)

So i tried to update the cohort definition. But the conceptId's appear to have been mapped to a lossy 
![image](https://github.com/OHDSI/Vocabulary-v5.0/assets/13936600/77ad8121-199c-454b-a3d6-9a4038f81ef4)

i.e. Spread to lymph node is a lossy mapping

Problem: without using non-standard concepts, i do not think i can reproduce my original cohort definition

**Comment by pbr6cornell** [comment]
I trust the vocab team's framing here @vladkorsik  @TinyRickC137  @dimshitc to explain how some of these concepts got 're-coordinated'.  But from what I can see the new vocab has multiple mappings from this now-non-standard concept, including 'is a' relation to standard 'Malignant neoplasm of abdomen', which I think would be helpful if the goal is to distinquish between concepts.

**Comment by gowthamrao** [comment]
>  including 'is a' relation to standard 'Malignant neoplasm of abdomen',

![image](https://github.com/OHDSI/Vocabulary-v5.0/assets/13936600/b76a416a-59dc-4722-993c-783596916167)


I would like to understand more about the re-coordination. e.g. 'Malignant neoplasm of abdomen' + 'Spread to lymph node' is semantically not the same as 'Metastatic malignant neoplasm to intra-abdominal lymph nodes'

Re-cordination in this case would clinically imply someone has a malignant neoplasm in the abdomen (maybe ovarian cancer) and there is spread to lymph node. That i think is different from what the coder may have originally intended when using source codes e.g. 'they have testicular cancer (which is not in abdomen) and have intraabdominal lymph nodes that is positive'

**Comment by cgreich** [comment]
Comments crossed.

Trying to explain it again:

1. The Condition is the primary malignant neoplasm, which is a combination of histology and topography, e.g. adenocarcinoma of the lung.
2. Lymph nodes are modifiers of the disease, which is the primary tumor. They develop over time. We decided to put them as Measurements, because that's how they most often appear (as a result of some test or imaging). The alternative was to pre-coordinate all modifiers with the condition e.g. creating adenocarcinoma of the lung with local lymph nodes affected. However, there are so many of them that could be pre-coordinated in parallel (metastases, grades, stages, genomic markers, invasion into neighboring structures) that we would have created a permutational explosion. 
3. You can have an explicit link between the Modifier (e.g. lymph node) and the primary, but that is of low value: If there is only one primary the lymph nodes belong to that one. If is more than one primary sometimes in the histology of the malignant tissue in the lymph node you can tell which one it is coming from. But in most cases you cannot. Which means, nobody knows.

**Comment by cgreich** [comment]
For phenotyping, I would generally **exclude any other malignant disease** except the one you are studying. Because otherwise all prognostic outcomes, complications, quality of life considerations and treatments and their effects are not comparable.

**Comment by gowthamrao** [comment]
Here is another example of loss of granularity

Previously standard code
![image](https://github.com/OHDSI/Vocabulary-v5.0/assets/13936600/f932bd13-9ef3-4467-b920-5b8f62ca9fb5)

it subsumed
![image](https://github.com/OHDSI/Vocabulary-v5.0/assets/13936600/ea2e5bd9-fad5-4598-b74d-9e76c94b44a1)


But now, this nice concept is Not standard and it is invalid(!)
![image](https://github.com/OHDSI/Vocabulary-v5.0/assets/13936600/7ee8ea4c-fb8d-4416-b829-4130ce7a2e47)

**Comment by gowthamrao** [comment]
https://athena.ohdsi.org/search-terms/terms/437677 has become non standard and maps to procedure domain that only has 'procedure was done' - and loosed 'abnormal'

**Comment by cgreich** [comment]
Hm. This is a completely different problem. Are you now going to list all standard concept changes in this Github issue? :)

But this case I am not sure why you would miss. First, it is very high level. "Diagnostic imaging of lung" is not something you will find in the data, it is really a classification. The only analytical use case I can think of is to use it to detect descendent records with actual procedures, such as x-rays, CTs etc.

Now, the "Abnormal findings" is indeed lost in the new mapping, And that could be improved. However, not as a pre-coordinated concept together with the procedure. Our procedure concepts do not have a result or outcome. Instead, we would record some lung Condition, like [Lung finding](https://athena.ohdsi.org/search-terms/terms/4115260). It also is **very** high level und unspecific, but the hierarchy would have a good chunk of problems in the lung. However, that list is not limited by what you can see in the imaging.

What's the use case? What kind of criterion are you building around it?

**Comment by gowthamrao** [comment]
> What's the use case? What kind of criterion are you building around it?

Here is an example: 'Diagnostics image abnormal' may be considered an index date misspecification of a lung cancer diagnosis, i.e. the hypothesis is some observation of lung nodule, patchy opacity etc, may give a more precise estimate of date of diagnosis. e.g. a person who had a lung nodule today and one month later had the lung cancer diagnosis - can the date the Diagnostics image was abnormal, be considered the earliest date the person is suspected to have lung cancer. This explains a few cases that get radiation/chemo prior to first diagnosis but after first 'diagnostics image abnormal'. This is not the most important use case for sure.


> Are you now going to list all standard concept changes in this Github issue? :)

I am happy to create separate issues if that is recommended.

**Comment by cgreich** [comment]
> I am happy to create separate issues if that is recommended.

I think these are Forum items. They are not Github issues (=errors). 

And I think you should come to the Onco WG. We are discussing issues like time of diagnosis regularly. It is not that simple.  Because we need to agree on this for the Episode definition (which in future you ought to use, rather than cobbling it together yourself). The candidates are first indication (a lump in imaging, a symptom), first diagnosis, first confirmed diagnosis (e.g. from a path lab), biopsy before first diagnosis. Folks are making good cases for either one.

Of course, it is a slightly silly discussion. Because no matter what you **declare** the date of the diagnosis is, the disease had developed over weeks and months before that, and we have no idea when it really started.

**Comment by gowthamrao** [comment]
> I think these are Forum items. They are not Github issues (=errors).

sounds good . overall, i have update about a 100 cohorts and found the vocabulary changes helpful. the concept set expressions have become simpler to work with. 

my purpose of this thread is to report issues i found unusual - i.e. i couldnt tell if they were by design or errors. I am ofcourse wanting to= learn and share my observations.

I will try to attend the Onco WG

**Comment by TinyRickC137** [comment]
> sounds good . overall, i have update about a 100 cohorts and found the vocabulary changes helpful. the concept set expressions have become simpler to work with.

Happy to hear it. We understand that some changes are very drastic, but the ideas behind them serve usability. 

2 cents regarding Procedures with abnormal results ([Colonoscopy abnormal](https://athena.ohdsi.org/search-terms/terms/4208962), [Imaging of lung abnormal](https://athena.ohdsi.org/search-terms/terms/40484489), etc.). It is indeed another issue, and the current [approach](https://github.com/OHDSI/Vocabulary-v5.0/issues/663#issuecomment-1213127684) has been to map to procedures and omit the results.
